kernel/src/lib/nuumshell/shell.cr

```diff
@@ -0,0 +1,66 @@
+require "./apps/*"
+
+class NuumShell
+  def initialize
+    @user = "root"
+    @pass = "root"
+    @login = false
+    @max_login = 3
```

Comment:
Nope, just the login procedure is restarting, and when we have more stuff (like a FS) a log can be created for this event